src/elements/content-sidebar/activity-feed/Avatar.js

```diff
@@ -38,9 +38,10 @@ class Avatar extends React.PureComponent<Props, State> {
      * @return {Promise<?string>} Promise which resolve with the avatar url string
      */
     getAvatarUrl(): Promise<?string> {
-        const { user, getAvatarUrl }: Props = this.props;
+        const { user = {}, getAvatarUrl }: Props = this.props;
+        const { avatar_url = null, id } = user;
 
-        const avatarPromise = getAvatarUrl ? getAvatarUrl(user.id) : Promise.resolve(user.avatar_url);
+        const avatarPromise = id && getAvatarUrl ? getAvatarUrl(`${id}`) : Promise.resolve(avatar_url);
```

Comment:
id is already a string according to the type?

Reply:
The type for this wrapper is User but it doesn't acknowledge some of the fields as nullable, and the underlying Avatar accepts numbers for the ID.

Changing the type has cascading consequences so we can follow up later.